curated_transformers/models/attention.py

```diff
@@ -191,6 +206,7 @@ def __init__(
         dropout_prob: float = 0.1,
         hidden_width: int = 768,
         num_attention_heads: int = 12,
+        qkv_mode: QkvMode = QkvMode.SEPARATE,
```

Comment:
I wouldn't specify a default value for this argument, this is something we want the implementation to always explicitly specify. The same is probably true for the `dropout_prob`, `hidden_width` and `num_attention_heads` arguments, where there is no good default.

Reply:
But this isn't `BertAttentionConfig`, but the constructor of the generic attention layer.

Reply:
Whoops! I didn't notice that :sweat_smile: Yeah, that makes sense :+1:

Reply:
Done.